src/models.ts

```diff
@@ -0,0 +1,62 @@
+import { FluxTableColumn, FluxTableMetaData, QueryApi } from '@influxdata/influxdb-client'
+
+type TableHead = FluxTableColumn[]
+type TableRow = string[]
+
+class TableResult {
```

Comment:
Does the existing client not provide a type for a table result? Or is this class a translation in representation to how the client returns data and how the rest of the plugin consumes data?

Reply:
Unfortunately, no, there really isn't. I was initially annoyed by this, but after using the API, it makes sense, because what you're doing with the results is just so domain specific when it comes to the client. The `{next, error, complete}` is a `FluxResultObserver`, and that's about the closest to a result you can get.